packages/open-next/src/build/patch/codePatcher.ts

```diff
@@ -51,27 +47,76 @@ export interface CodePatcher {
   patches: IndividualPatch | VersionedField<IndividualPatch>[];
 }
 
+export function parseVersions(versions?: Versions): {
+  before?: string;
+  after?: string;
+} {
+  if (!versions) {
+    return {};
+  }
+  // We need to use regex to extract the versions
+  const versionRegex = /([<>]=?)([0-9]+\.[0-9]+\.[0-9]+)/g;
```

Comment:
- `=` is not optional
- using `\d` for digits

```suggestion
  const versionRegex = /([<>]=)(\d+\.\d+\.\d+)/g;
```